src/beatmap.c

```diff
@@ -185,6 +185,28 @@ static void parse_one_hit(char *line, struct oshu_hit **hit)
 }
 
 /**
+ * Parse #oshu_hit::combo and #oshu_hit::combo_seq of a single #oshu_hit
+ */
+static void parse_hit_combo(struct oshu_hit *previous, struct oshu_hit *hit)
+{
+	int new_combo = hit->type & OSHU_HIT_NEW_COMBO;
+	int skip_combo = (hit->type & OSHU_HIT_COMBO_MASK) >> 4;
```

Comment:
By the way, I'm not applying the remark above, because while this variable is used only once, it is used in a complex expression.